test/specs/smooch.spec.js

```diff
@@ -97,9 +105,11 @@ describe('Smooch', () => {
         });
 
         getItemStub = sandbox.stub();
+        removeItemStub = sandbox.stub();
         SmoochRewire('storage', {
```

Comment:
https://github.com/smooch/smooch-js/blob/f/compat-expansion/test/index.js#L12

Go be on vacation